OpenSim/Simulation/Model/BushingForce.h

```diff
@@ -80,15 +75,28 @@ OpenSim_DECLARE_CONCRETE_OBJECT(BushingForce, Force);
 //==============================================================================
 // PUBLIC METHODS
 //==============================================================================
-    /** Default constructor leaves bodies unspecified, sets the bushing frames
+    /** Default constructor leaves frames unspecified, sets the bushing frames
     to be at their body origins, and sets all bushing parameters to zero. **/
     BushingForce();
-    /** This constructor provides everything needed to create a real 
-    bushing. See property declarations for more information. **/
-    BushingForce(const std::string& body1Name, 
+
+    /** Construct the BushingForce given the offset frames that it tries to
+    align according to the physical properties of the bushing.
+    See property declarations for more information. **/
+    BushingForce(const PhysicalOffsetFrame& frame1,
```

Comment:
The BushingForce connects two PhysicalFrames. The purpose of a PhysicalOffsetFrame is to offset a PhysicalFrame to which it connects. I am using the PhysicalOffsetFrame to make my task of connecting anywhere on any PhysicalFrame easier (in my opinion). If you ask a BushingForce what connectors it has they are PhysicalFrames, not the PhysicalOffsetFrames but their connectors. I was able to get a connector<PhysicalFrame> and generally useful offset properties for "free" by making the PhysicalOffsetFrames an attribute  (property) of the BushingForce. I thought it was a nice feature to use a Component as a property in this way.